settings

```diff
@@ -20,18 +20,20 @@ load_settings() {
   # Load settings per release
   # Expected: FULLGPGKEY
   # Optional: FULLVERSION
-  . "$RELEASEDIR/settings"
+  if [[ -f "${RELEASEDIR}/settings" ]]; then
+    . "$RELEASEDIR/settings"
+  fi
 }
 
 # Prepare settings file
 if [[ ! -d "$RELEASEDIR" ]] ; then
 	mkdir -p "$RELEASEDIR"
 	if [[ $PROJECT == katello ]] ; then
 		echo "FULLVERSION='${VERSION}.0.rc1'" > "$RELEASEDIR/settings"
-	else
+	elif [[ $PROJECT == foreman ]]; then
 		echo "FULLVERSION='${VERSION}.0-rc1'" > "$RELEASEDIR/settings"
 	fi
-	if [[ $PROJECT != foreman ]] && [[ $FOREMAN_VERSION != "none" ]] ; then
+	if [[ $PROJECT != foreman ]] && [[ $FOREMAN_VERSION != "none" ]] && [[ -n "$FOREMAN_VERSION" ]] ; then
```

Comment:
I added this so there was an explicit empty value as well because it is expected to be set.

Reply:
With `katello` I want `FOREMAN_VERSION=` in the generated file, regardless of whether it's empty or not. I didn't think of `plugins` where it's special.

Reply:
How about limiting it to just katello?
```suggestion
	if [[ $PROJECT == katello ]] ; then
```

And then move it into the existing katello block.

Reply:
Yeah, that seems to do it, thanks!